mail/api_test.py

```diff
@@ -150,18 +153,43 @@ def test_send_batch_chunk(self, mock_post):
             )
 
     @override_settings(MAILGUN_RECIPIENT_OVERRIDE=None)
-    def test_send_individual_email(self, mock_post):
+    @data(None, 'Tester')
+    def test_send_individual_email(self, sender_name, mock_post):
         """
         Test that MailgunClient.send_individual_email() sends an individual message
         """
-        MailgunClient.send_individual_email('email subject', 'email body', 'a@example.com')
+        MailgunClient.send_individual_email('email subject', 'email body', 'a@example.com', sender_name=sender_name)
```

Comment:
maybe call the method with the keywords, so it is a bit more clear what's going on